_docs-sources/refarch/configuration/index.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started
+
+The Gruntwork Reference Architecture allows you to configure key aspects to your needs. Before you receive your deployed Reference Architecture, you will:
+1. **Configure** your choice of your primary AWS region, database and compute flavors, domain names and more via a pull request
+2. **Iterate** on the configuration in your pull request in response to Gruntwork preflight checks that spot blocking issues and ensure your deployment is ready to commence
+3. **Merge** your pull request after all check pass. Merging will automatically commence your Reference Architecture deployment
+4. **Wait** until Gruntwork has successfully completed your deployment. You'll receive an automated email indicating your deployment is complete
+
+## Configure your Reference Architecture for deployment
+
+Your Reference Architecture configuration lives in your `infrastructure-live` repository on GitHub. Within your `infrastructure-live` repository, the `reference-architecture-form.yml` file defines all of your specific selections, domain names, AWS account IDs, etc.
```

Comment:
Do you think it's worth clarifying that this is referring to a repo we've created for them in the gruntwork-clients organization? Something like `...configuration lives in the infrastructure-live repository we created for you in the gruntwork-clients Github organization...` Maybe with a call to action for if they don't have access linking to the portal

Reply:
Added in the requirements section